app/functions/formatDates.ts

```diff
@@ -7,7 +7,7 @@ const FORMAT_TYPE = {
   seconds: 'MMM D, YYYY hh:mm:ss A (z)',
   minutes: 'MMM D, YYYY hh:mm A (z)',
   days_numbered: 'DD-MM-YYYY',
-  days_string: 'MMM Do, YYYY'
+  days_string: 'MMMM Do, YYYY'
```

Comment:
The only place this is currently used is on the homepage/LoginRegistrationButtons and in `/admin/reporting-years`, which makes it low-risk to update.

I'm pretty certain the entire date ("July" as opposed to "Jul") used to be spelled out before the deadline was hidden, so it must have gotten changed somewhere along the way.